builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -0,0 +1,240 @@
+package vsphere
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"golang.org/x/net/context"
+)
+
+func TestAccVSphereVirtualMachine_basic(t *testing.T) {
+	var vm virtualMachine
+	datacenter := os.Getenv("VSPHERE_DATACENTER")
```

Comment:
These testa are quite complex to start with. A documented example of expected settings would be helpful.